lib/watir-webdriver/elements/element.rb

```diff
@@ -570,6 +585,15 @@ def assert_is_element(obj)
       end
     end
 
+    def element_call
+      yield
+    rescue Selenium::WebDriver::Error::ObsoleteElementError, UnknownObjectException
+      raise UnknownObjectException unless Watir.always_locate?
```

Comment:
A few points here:

1. In what cases will the yield raise an `UnknownObjectException`? Is that rescue actually necessary?
2. It's always better to rescue `StaleElementReferenceError`, which is the [new name](https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/webdriver/common/error.rb#L191) for `ObsoleteElementError`. 
3. I don't think it's ideal to raise `UnknownObjectException` if `Watir.always_locate?` is false. We're "erasing" both the error message and the specificity of the `StaleElementReferenceError`, so the user has no way of knowing whether this was caused by an element going stale or if it never existed. This will break backwards compatibility for people (like me) who have code that relies on stale element exceptions. I think we should either (a) re-raise the original exception or (b) create a new Watir error that inherits from `StaleElementReferenceError` - thus ensuring backwards compatibility without leaking WebDriver errors.